### ISSUE TITLE
Add charts to personal results page

### ISSUE THREAD

**Issue by rogerlejeune** [xabikos/BrazilWC2014#45]
Add a line graph to see progress per match. 

And as an extra nice to have: compare your progress with 1 (or more) other player(s).

**Comment by xabikos** [comment]
Maybe during the weekend if it's not too late already :8ball:

**Comment by rogerlejeune** [comment]
Yes, I was thinking of a doughnut graph (since you are in the land of doughnuts, not donkeys) where are the slices represent your total score of that round. However, it doesn't work well yet in the design and I think I should come up with something better, or have you got a good suggestion?
